spectests/structs_encoding.go

```diff
@@ -1795,11 +1807,17 @@ func (a *AttesterSlashing) UnmarshalSSZ(buf []byte) error {
 	if o0 = ssz.ReadOffset(buf[0:4]); o0 > size {
 		return ssz.ErrOffset
 	}
+	if o0 < 8 {
```

Comment:
Can you merge this check with the one in line 1807? Both are part of the same offset check

Reply:
See the comment in unmarshal.go for a possible solution